container-startup

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash -e
+# Startup helper for pcw. This file makes the handling with pcw within containers easier
+# See usage() for a usage summary
+
+CMD=$1
+shift
+
+function usage() {
+	echo "pcw startup helper"
+	echo "Usage: $0 COMMAND"
+	echo "COMMANDS:"
+	echo "   help                    Print help message"
+	echo "   run                     Standart publiccloud-watcher run procedure"
+	echo "   migrate                 Run database migration"
+	echo "   createsuperuser         Create superuser. Requires --email and --username argument"
+	echo ""
+	echo "All other commands will be passed as arguments to 'manage.py'"
+}
+
+case "$CMD" in
+  "")
+  	usage
+  	exit 1
+  	;;
+  "help")
+  	usage
+  	exit 0
+  	;;
+  "-h")
+  	usage
+  	exit 0
+  	;;
+  "--help")
+  	usage
+  	exit 0
+  	;;
+  "run")
+    # Ensure the database directory exists
+    mkdir -p /data/db
```

Comment:
`createsuperuser` can be executed via `podman exec`. If this is necessary, it will be documented in the publiccloud-qa-suse-de repository. This command is the reason, why I created `/pcw/pcw` in the container to make it easier to execute such commands (previous command).

I have no idea what `collectstatic` does and if we need it.

Reply:
> createsuperuser can be executed via podman exec

why not to keep everything in one place as it was before ? 

> I have no idea what collectstatic does and if we need it.

we need to clarify this

Reply:
We in my opinion simply don't have to do the migrations every time.

Reply:
not sure what do you mean when you saying "every time" . main idea which we need to keep in mind - this approach suppose to work out-of-the-box on clean machine not having anything . Our current state in `vault-docker` project there is no need to run any commands outside of bash scripts . I don't care if we will migrate this into ansible/salt/you-name-it but main thing which I would like to keep - I fire one line in terminal and than some magic install everything what I need and starting all needed containers . And this command suppose to work on fresh installed system

Reply:
if instead we will have approach where there will be well documented flow which force me to execute set of commands one by one I would treat that as step back